### ISSUE TITLE
about fcn  train

### ISSUE THREAD

**Issue by lius0-0** [jsgaobiao/FCN.tensorflow#1]
visible  image is  black  in  shekkizh/fcn   
class = 151 when i change it and datasets  the loss is NAN   what should i do ....   thank  you,if you read it....i  am so  stupid...
